src/easyscience/Constraints.py

```diff
@@ -60,11 +60,11 @@ def __init__(
             # Test if dependent is a parameter or a descriptor.
             # We can not import `Parameter`, so......
             if dependent_obj.__class__.__name__ == 'Parameter':
-                if not dependent_obj.enabled:
+                if not dependent_obj.independent:
                     raise AssertionError('A dependent object needs to be initially enabled.')
```

Comment:
```suggestion
                    raise AssertionError('A dependent object needs to be initially independent.')
```
Or does semantically make sense? What did it mean for a dependent_obj to be `enabled`?